### PR TITLE
App initialization with redux

### PR DESCRIPTION
**Pull request by alvikalvik**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -5,31 +5,38 @@ import 'normalize.css';
...
-} from './constants/constants';
-import { weatherAPI } from './services/dataService';
 import ContentWrapperContainer from './containers/ContentWrapperContainer';
-import { useSelector } from 'react-redux';
+import { useDispatch, useSelector } from 'react-redux';
+import { useEffect } from 'react';
+import { getToken } from './actions/ServerApiActions';
+import { getPosition } from './actions/GeoDetectionActions';
+
+function useInitialize() {
```
Comment (anton-vlasik) [inline src/App.jsx]: I'd suggest to move your custom hooks to a dedicated folder and module

File: src/actions/ServerApiActions.js [hunk 2]
```diff
@@ -4,23 +4,50 @@ import {
...
-export const setIsFetchingError = fetchingError => ({
+export const setFetchingError = fetchingError => ({
   type: SET_FETCHING_ERROR,
-  payload: { fetchingError }
+  payload: {
+    fetchingError
+  }
 });
+
+export const getToken = () => dispatch => {
```
Comment (anton-vlasik) [inline src/actions/ServerApiActions.js:38]: OPTIONAL: you could omit additional closure: 
`export const getToken = dispatch => {`

File: src/App.jsx (hunk 1 above)
Comment (alvikalvik) [reply src/App.jsx]: ok. Moved to /hooks/initialize.js
Thanks

File: src/actions/ServerApiActions.js (hunk 2 above)
Comment (alvikalvik) [reply src/actions/ServerApiActions.js:38]: I see. I prefer to leave this closure for consistency as we discussed in chat.

**Review by anton-vlasik — APPROVED** [review]

**Review by ximet — APPROVED** [review]

**Pull request merged**